test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMessageReaderSettingsTests.cs

```diff
@@ -205,6 +205,9 @@ private void CompareMessageReaderSettings(ODataMessageReaderSettings expected, O
             Assert.True(expected.MessageQuotas.MaxOperationsPerChangeset == actual.MessageQuotas.MaxOperationsPerChangeset, "MaxOperationsPerChangeset does not match");
             Assert.True(expected.MessageQuotas.MaxNestingDepth == actual.MessageQuotas.MaxNestingDepth, "MaxNestingDepth does not match");
             Assert.True(expected.MessageQuotas.MaxReceivedMessageSize == actual.MessageQuotas.MaxReceivedMessageSize, "MaxMessageSize does not match");
+
+            var differences = ValidationHelper.GetDifferences<ODataMessageReaderSettings>(expected, actual);
+            Assert.True(differences.Count == 0, String.Join(",", differences));      

```

Comment:
If you add the "GetDifferences<T>" function call, how about the codes from Line 191 to Line 207?

Reply:
Didn't want to change the existing code. Existing lines will function as such

Reply:
That's my concern. It seems we verify the same twice.

---
In reply to: [348177998](https://github.com/OData/odata.net/pull/1595#discussion_r348177998) [](ancestors = 348177998)